include/wlr/xwayland.h

```diff
@@ -0,0 +1,41 @@
+#ifndef _WLR_XWAYLAND_H
+#define _WLR_XWAYLAND_H
+#include <time.h>
+#include <sys/types.h>
+#include <stdbool.h>
+#include <wlr/types/wlr_compositor.h>
+#include <xcb/xcb.h>
+
+struct wlr_xwm;
+
+struct wlr_xwayland {
+	pid_t pid;
+	int display;
+	int x_fd[2], wl_fd[2], wm_fd[2];
+	struct wl_client *client;
+	struct wl_display *wl_display;
+	struct wlr_compositor *compositor;
+	time_t server_start;
+
+	struct wl_event_source *sigusr1_source;
+	struct wlr_xwm *xwm;
+	struct wl_list displayable_windows;
+};
+
+struct wlr_x11_window {
+	xcb_window_t window_id;
+	uint32_t surface_id;
+	struct wl_list link;
+
+	struct wl_resource *surface;
+	struct wl_listener surface_destroy_listener;
+	int16_t x, y;
+	uint16_t width, height;
```

Comment:
Actually not using x/y/width/height there, I thought I would need them but didn't.
Not sure if I will later yet, but these come from Xwayland on the wayland client side anyway so probably safe to remove again unless we need some notifications.